packages/app/package.json

```diff
@@ -17,7 +17,7 @@
         "@json2csv/node": "^7.0.1",
         "@rjsf/core": "^5.16.1",
         "@rjsf/utils": "^5.16.1",
-        "@rjsf/validator-ajv8": "^5.16.1",
+        "@rjsf/validator-ajv8": "^6.0.0-alpha.0",
```

Comment:
Typically running alpha versions is not a good idea...however this resolves a major performance issues with RJSF v5